examples/custom_action/example_server/swagger_server/controllers/default_controller.py

```diff
@@ -0,0 +1,37 @@
+import connexion
+
+from ..models.data import Data  # noqa: E501
```

Comment:
Why do we need lint exemptions in examples? Can we just exempt this whole directory?

Reply:
This code is autogenerated by Swagger